README.md

```
@@ -16,38 +18,71 @@ It's impossible to talk about *"mockingbird"* without remembering the song [Mock
+
+
+``` clj
+  ($ button {:class "special-class-i-want-to-add"
+             :theme :mockingbird
+             :size :sm 
+             :label "My-special-label"})
+
+```
+As you can see, our components have specific built-in styles for you. You can see a handful tutorial on the parameters for styling the components [here](docs/params.md)
+
+# Testing Locally 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)